sphinx/ext/napoleon/__init__.py

```diff
@@ -257,6 +258,9 @@ def __unicode__(self):
         section. If the entry is a tuple/list/indexed container, the first entry
         is the name of the section, the second is the section key to emulate.
 
+    napoleon_google_attr_annotations : :obj:`bool` (Defaults to False)
```

Comment:
Is there any reason to disable this feature by default?

Reply:
I was waiting on some sort of approval of the feature before enabling it by default. It is now enabled